HolyshiftService/JwtService.cs

```diff
@@ -0,0 +1,96 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+public class JwtService
+{
+    private readonly byte[] _keyBytes = new byte[32];
+    private readonly string _issuer;
+    private readonly string _audience;
+    private readonly int _accessTokenExpirationMinutes;
+    private readonly int _refreshTokenExpirationDays;
+    private readonly ILogger<JwtService> _logger;
+
+    public JwtService(IConfiguration configuration, ILogger<JwtService> logger)
```

Comment:
Instead of injecting `IConfiguration` inject `IOptions<AuthConfig>`. Similar to what you did with the DB Config.

> In theory we can have a high level config class which contains all the config sub classes. Currently we don't need it but keep that in mind as we might do that in the future.